src/query/service/src/pipelines/processors/transforms/window/transform_window.rs

```diff
@@ -652,6 +647,79 @@ impl<T: Number> TransformWindow<T> {
         }
         false
     }
+
+    #[inline]
+    fn find_nth_non_null_value(
```

Comment:
the function accept a arg named `ignore_null`, so it's name should not be  non_null.